CHANGES.md

```diff
@@ -1,12 +1,20 @@
 Libplanet changelog
 ===================
 
-Version 0.21.2
+Version 0.21.3
 --------------
 
 To be released.
 
 
+Version 0.21.2
+--------------
+
+Released on November 25, 2021.
+
+-  `planet apv query` use `NetMQTransport.QueryAppProtocolVersion()`.
```

Comment:
```suggestion
-  `planet apv query` command became to communicate with `NetMQTransport` 
    instead of `TcpTransport`.   [[#1618]]
    
[#1618]: https://github.com/planetarium/libplanet/pull/1618
```